bentoml/xgboost.py

```diff
@@ -1,14 +1,19 @@
 import os
-import sys
 import typing as t
 
 import bentoml._internal.constants as _const
 import bentoml._internal.models.store as _stores
 
-from ._internal.models.base import JSON_EXTENSION, MODEL_NAMESPACE, Model
+from ._internal.models import (
+    JSON_EXT,
+    LOAD_INIT_DOCS,
+    SAVE_INIT_DOCS,
+    SAVE_NAMESPACE,
+    SAVE_RETURNS_DOCS,
+)
 from ._internal.service.runner import Runner
-from ._internal.types import GenericDictType, PathType
-from ._internal.utils import LazyLoader
+from ._internal.types import GenericDictType
+from ._internal.utils import LazyLoader, init_docstrings, returns_docstrings
```

Comment:
let's put init_docstrings, returns_docstrings in the `models` namespace?

Reply:
sounds good